packages/core/test/cross-chain-oracle/GovernorSpoke.js

```diff
@@ -25,16 +28,37 @@ describe("GovernorSpoke.js", async () => {
     // Deploy contract that Governor should be able to delegate execution to:
     addressWhitelist = await AddressWhitelist.new().send({ from: owner });
     await addressWhitelist.methods.transferOwnership(governor.options.address).send({ from: owner });
+
+    identifierWhitelist = await IdentifierWhitelist.new().send({ from: owner });
+    await identifierWhitelist.methods.transferOwnership(governor.options.address).send({ from: owner });
   });
   it("Constructor", async function () {
     const setChildMessengerEvents = await governor.getPastEvents("SetChildMessenger", { fromBlock: 0 });
     assert.equal(setChildMessengerEvents.length, 1);
     assert.equal(setChildMessengerEvents[0].returnValues.childMessenger, messenger);
   });
+
   it("Can delegate call if called by Messenger", async function () {
-    let targetAddress = addressWhitelist.options.address;
-    let inputDataBytes = addressWhitelist.methods.addToWhitelist(messenger).encodeABI();
-    let messageBytes = web3.eth.abi.encodeParameters(["address", "bytes"], [targetAddress, inputDataBytes]);
+    const calls = [
```

Comment:
this is really not too bad! having to do the `web3.eth.abi.encodeParameters` on this array of objects (struct) is a bit painful but nothing we cant abstract away in a script :)